cluster-provision/centos8/scripts/kernel.args

```diff
@@ -1 +1 @@
-root=/dev/vda1 ro no_timer_check console=tty0 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 elevator=noop spectre_v2=off nopti hugepagesz=2M hugepages=64 intel_iommu=on modprobe.blacklist=nouveau
\ No newline at end of file
+root=/dev/vda1 ro no_timer_check console=tty0 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 elevator=noop spectre_v2=off nopti hugepagesz=2M intel_iommu=on modprobe.blacklist=nouveau
```

Comment:
Since we already take out hugepages, let's also move out the `hugepagesz=2M` setting from here and move it up to `gocli`. We have thenall parts which one would potentially want to modify removed from here.

Reply:
Sounds good. I'll parametrize it as well. Since I don't know which kind of environments it will run (x86, arm), I will not check the value of the page size to avoid locking it to certain platforms (x86). What do you think?

Reply:
> Sounds good. I'll parametrize it as well. Since I don't know which kind of environments it will run (x86, arm), I will not check the value of the page size to avoid locking it to certain platforms (x86). What do you think?

How about adding the following flag: `--hugepages-2m` instead of an extra parameter?

Reply:
That would be easier indeed. I'll just add the flag and set the default value as true for now.

This means we're skipping support for `1G` and ppc64 arch page sizes, right?  [(ppc64 uses 4k and 16M)](https://www.kernel.org/doc/html/latest/admin-guide/mm/hugetlbpage.html).

Reply:
I meant converging all `2m` settings into one single parameter and not adding a boolean. So that `--hugepages-2m=64` would add both kernel parameters. But happy if it works for you as is.

Reply:
Makes sense and it looks cleaner (1 flag vs 2 flags). I'll raise a new PR to adjust to this suggestion. Should be painless :smile:

Reply:
We will still need another parameter if we will need to support 1G pages, won't we?

Reply:
That's true, but I wonder if that case is coming anytime soon (disclaimer: I don't know). If there is no plans in the horizon to support other than 2MB sizes for functional testing, it might be a case of over engineering.
Otherwise, I'm happy to leave the structure in the code to support other sizes :smile: